pallets/file-system/src/lib.rs

```diff
@@ -840,7 +855,51 @@ pub mod pallet {
             Ok(())
         }
 
+        /// Executed by a SP to stop storing a file from an insolvent user.
+        ///
+        /// This is used when a user has become insolvent and the SP needs to stop storing the files of that user, since
+        /// it won't be getting paid for it anymore.
+        /// The validations are similar to the ones in the `bsp_request_stop_storing` and `bsp_confirm_stop_storing` extrinsics, but the SP doesn't need to
+        /// wait for a minimum amount of blocks to confirm to stop storing the file nor it has to be a BSP.
         #[pallet::call_index(9)]
+        #[pallet::weight(10_000 + T::DbWeight::get().reads_writes(1,1).ref_time())]
+        pub fn stop_storing_for_insolvent_user(
```

Comment:
There should be some testing for this extrinsic.

Reply:
Added!